README.md

```diff
@@ -249,10 +247,10 @@ jobs:
 
 ### `configure` Action Inputs
 
-| Name                    | Default | Environment variable       | Description                                              |
-| ----------------------- | ------- | -------------------------- | -------------------------------------------------------- |
-| `connect-host`          |         | `OP_CONNECT_HOST`          | Your 1Password Connect instance URL                      |
-| `connect-token`         |         | `OP_CONNECT_TOKEN`         | Token to authenticate to your 1Password Connect instance |
+| Name            | Default | Environment variable | Description                                              |
+| --------------- | ------- | -------------------- | -------------------------------------------------------- |
+| `connect-host`  |         | `OP_CONNECT_HOST`    | Your 1Password Connect instance URL                      |
+| `connect-token` |         | `OP_CONNECT_TOKEN`   | Token to authenticate to your 1Password Connect instance |
```

Comment:
If there's no defaults in this table, maybe we don't need this column?

Reply:
They are undefined indeed.

The way the action works is that is you provide a value to e.g. `connect-host`, then the `configure` action will make it available for the following steps as an environment variable. The default is the empty string and in this case, the action will do nothing.

Therefore, we can safely remove the `Default` column since it brings no additional value.